packages/react-divider/src/components/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { Divider } from './Divider';
+import * as renderer from 'react-test-renderer';
+import { ReactWrapper } from 'enzyme';
+import { isConformant } from '../../common/isConformant';
+
+describe('Divider', () => {
+  isConformant({
+    Component: Divider,
+    displayName: 'Divider',
+  });
+
+  let wrapper: ReactWrapper | undefined;
+
+  afterEach(() => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = undefined;
+    }
+  });
+
+  /**
+   * Note: see more visual regression tests for Divider in /apps/vr-tests.
+   */
+  it('renders a default divider', () => {
+    const component = renderer.create(<Divider />);
```

Comment:
I would suggest to use `@testing/library-react` currently older components use `enzyme` (which AFAIK we're moving away from) but this is probably the first time I've seen a package using straight up test renderer, let's try to be consistent in the repo

Reply:
Lets have a separate PR to move all converged packages to testing/library-react.